### DISCUSSION TITLE
Error handling

### DISCUSSION THREAD

**Discussion by fethallaheth** [foundry-rs/foundry#5202] [Help]
i  cannot connect foundry with ganache
 ; 

i use windows-64  bash ||

Error
'''$ forge create SimpleStorage --rpc-url http://127.0.0.1:7545/ --interactive
[⠑] Compiling...
No files changed, compilation skipped
Error:
The handle is invalid. (os error 6)'''

even if i do --public-key the same error

**Comment by alfheimrShiven** [comment]
> i cannot connect foundry with ganache ;
> 
> i use windows-64 bash ||
> 
> Error '''$ forge create SimpleStorage --rpc-url http://127.0.0.1:7545/ --interactive [⠑] Compiling... No files changed, compilation skipped Error: The handle is invalid. (os error 6)'''
> 
> even if i do --public-key the same error

Hey @fethallaheth try to use the private key:
```forge create SimpleStorage --rpc-url http://127.0.0.1:7545 --private-key {yourPrivateKey}```
